openssl.nuspec

```diff
@@ -1,37 +1,24 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek
      uppercase omega letter enclosed in quotation marks,
-	 you should use an editor that supports UTF-8, not this one. -->
+     you should use an editor that supports UTF-8, not this one. -->
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>openssl</id>
     <title>OpenSSL (for software developers)</title>
-    <version>1.0.2.20150731</version>
-    <authors>Alex Willmer</authors>
-    <owners>Alex Willmer</owners>
+    <version>1.0.2.20160504</version>
```

Comment:
I can't remember if I used a date suffix because it's recommended/required, or if it just seemed a good idea at the time. Please feel free to use this versioning scheme or any other that seems suitable.